.github/workflows/ci.yml

```diff
@@ -22,12 +22,8 @@ jobs:
       fail-fast: false
       matrix:
         os: ["ubuntu-latest", "macos-latest", "windows-latest"]
-        rust: ["stable", "1.83"]
+        rust: ["stable"]
         flags: ["", "--all-features"]
-        exclude:
-          # Skip because some features have higher MSRV.
-          - rust: "1.83" # MSRV
-            flags: "--all-features"
```

Comment:
this is not right, you bump the msrv not remove the test that checks compatibility :D

Reply:
:melting_face:

Reply:
redone, pls check